packages/ecs-agent/0005-bottlerocket-change-execcmd-directories-for-Bottlero.patch

```diff
@@ -1,35 +1,36 @@
-From cef5037351f2b214b9239d59dec7a83a0db75b0c Mon Sep 17 00:00:00 2001
-From: Arnaldo Garcia Rincon <agarrcia@amazon.com>
-Date: Wed, 3 May 2023 18:23:40 +0000
+From f8384b98acd66714b2bd4cbf1da3ba3308e2754c Mon Sep 17 00:00:00 2001
+From: Yutong Sun <yutongsu@amazon.com>
```

Comment:
Did you re-create the patch? If you applied the patch in a fork, it should have kept authorship. Ideally, you should keep the original authorship and add your `Singed-off` footer in the commit message of the patch.